src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -30,6 +30,16 @@ Optional<object> IExpression.ConstantValue
         SyntaxNode IOperation.Syntax => this.Syntax;
         
         protected virtual OperationKind ExpressionKind => OperationKind.None;
+
+        public virtual void Accept(IOperationVisitor visitor)
+        {
+            throw ExceptionUtilities.Unreachable;
+        }
+
+        public virtual TResult Accept<TResult>(IOperationVisitor<TResult> visitor)
+        {
+            throw ExceptionUtilities.Unreachable;
+        }
```

Comment:
In that case, all the `BoundExpression` and `BoundStatement` will have to implement it, including those not related to `IOperation`. It will guarantee us not mistakenly miss any implementation but that's a lot of nodes...